tests/migration/migration.go

```diff
@@ -269,12 +270,28 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 		})
 		Context("[Serial] with bandwidth limitations", Serial, func() {
 
-			var repeatedlyMigrateWithBandwidthLimitation = func(vmi *v1.VirtualMachineInstance, bandwidth string, repeat int) time.Duration {
+			updateMigrationPolicyBandwidth := func(migrationPolicy *v1alpha1.MigrationPolicy, bandwidth resource.Quantity) {
+				migrationPolicy, err = virtClient.MigrationPolicy().Get(context.Background(), migrationPolicy.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				if policyBandwidth := migrationPolicy.Spec.BandwidthPerMigration; policyBandwidth != nil && policyBandwidth.Equal(bandwidth) {
+					return
+				}
+
+				patchPayload, err := patch.New(
+					patch.WithAdd("/spec/bandwidthPerMigration", pointer.P(bandwidth)),
+				).GeneratePayload()
+				Expect(err).ToNot(HaveOccurred())
+
+				migrationPolicy, err = virtClient.MigrationPolicy().Patch(context.Background(), migrationPolicy.Name, types.JSONPatchType, patchPayload, metav1.PatchOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(migrationPolicy.Spec.BandwidthPerMigration.Equal(bandwidth)).To(BeTrue(),
```

Comment:
I don't think we need this. Here it is like we are not trusting in the `Patch` API.
If something goes wrong the `Expect(err).ToNot(HaveOccurred())` will catch it.